src/CONST.ts

```diff
@@ -4822,6 +4822,9 @@ const CONST = {
         DESC: 'desc',
     },
 
+    PAGINATION_START_ID: '-1',
+    PAGINATION_END_ID: '-2',
```

Comment:
@janicduplessis I'm seeing where this value is _read_, but I'm never seeing it _set_ anywhere? What am I missing?

Reply:
We currently do not set the END one since we don't have a reliable way to know that we reached the end. My plan was to use it when we implement pagination info returned from the backend.

Reply:
We could check the report type for CREATED, which is what we do in ReportScreen to avoid calling GetOlderActions more.